test/servinge2e/encryption/helper.go

```diff
@@ -0,0 +1,41 @@
+package encryption
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/openshift-knative/serverless-operator/test"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"knative.dev/networking/pkg/certificates"
+)
+
+// this needs to match the ClusterIssuer in hack/lib/certmanager_resources/serving-ca-issuer.yaml.
```

Comment:
s/this/This

Reply:
@skonto as this is just cosmetics, can we skip this to get this merged?